lighthouse-core/runner.js

```diff
@@ -258,7 +258,7 @@ class Runner {
     const computedArtifacts = {};
     require('fs').readdirSync(__dirname + '/gather/computed').forEach(function(filename) {
       // Skip base class.
-      if (filename === 'computed-artifact.js') return;
+      if (filename === 'computed-artifact.js' || filename.startsWith('dependency-graph')) return;
```

Comment:
can this be 'dependency-graph/' maybe? It took me a while to realize this was talking about the directory path rather than the start of a filename

Reply:
oh ya sure that's clearer 👍